build/test-scripts/ios-uitest-run.sh

```diff
@@ -82,8 +89,7 @@ export UNO_TESTS_LOCAL_TESTS_FILE=$BUILD_SOURCESDIRECTORY/src/SamplesApp/Samples
 export UNO_UITEST_BENCHMARKS_PATH=$BUILD_ARTIFACTSTAGINGDIRECTORY/benchmarks/ios-automated
 export UNO_UITEST_RUNTIMETESTS_RESULTS_FILE_PATH=$BUILD_SOURCESDIRECTORY/build/RuntimeTestResults-ios-automated.xml
 
-if [ $(wc -l < "$UNO_TESTS_FAILED_LIST") -eq 1 ];
-then
+if [ $(wc -l < "$UNO_TESTS_FAILED_LIST") -eq 1 ] && [ "$IGNORE_RERUN_FILE" != "true" ]; then
```

Comment:
We could probably use default values like this for the `IGNORE_RERUN_FILE` variable:

```suggestion
IGNORE_RERUN_FILE="${IGNORE_RERUN_FILE:=false}"

if [ $(wc -l < "$UNO_TESTS_FAILED_LIST") -eq 1 ] && [ "$IGNORE_RERUN_FILE" != "true" ]; then
```